src/test/java/com/epam/jamp2/service/impl/CalculationCommandExecutionServiceTest.java

```diff
@@ -0,0 +1,158 @@
+                    }
+
+                    @Override
+                    public void cancel() {
+                    }
+
+                    @Override
+                    public boolean isCanceled() {
+                        return false;
+                    }
+
+                    @Override
+                    public Call<FixerioResponse> clone() {
+                        return null;
+                    }
+
+                    @Override
+                    public Request request() {
+                        return null;
+                    }
+                });
+
+        //Inject mocked service to related class
+        calculationCommandExecutionService = new CalculationCommandExecutionServiceImpl();
+        calculationCommandExecutionService.fxRatesService = fxRatesServiceImpl;
+    }
+
+    @Test
+    public void testConversionUnnecessary() throws IOException, UnknownCurrencyException {
+        CalculationCommand calculationCommand = new CalculationCommand(
+                new Value("usd", BigDecimal.valueOf(100)),
+                new Value("usd", BigDecimal.valueOf(5)),
+                Operation.ADD, "usd");
+        assertThat(calculationCommandExecutionService.calculate(calculationCommand).getValue(),
+                greaterThan(BigDecimal.ZERO));
+        calculationCommand = new CalculationCommand(
+                new Value("cny", BigDecimal.valueOf(100)),
+                new Value("cny", BigDecimal.valueOf(5)),
+                Operation.SUBTRACT, "cny");
+        assertThat(calculationCommandExecutionService.calculate(calculationCommand).getValue(),
+                greaterThan(BigDecimal.ZERO));
+        calculationCommand = new CalculationCommand(
+                new Value("hkd", BigDecimal.valueOf(100)),
+                new Value("hkd", BigDecimal.valueOf(5)),
+                Operation.MULTIPLY, "hkd");
+        assertThat(calculationCommandExecutionService.calculate(calculationCommand).getValue(),
+                greaterThan(BigDecimal.ZERO));
+        calculationCommand = new CalculationCommand(
+                new Value("jpy", BigDecimal.valueOf(100)),
+                new Value("jpy", BigDecimal.valueOf(5)),
+                Operation.DIVIDE, "jpy");
+        assertThat(calculationCommandExecutionService.calculate(calculationCommand).getValue(),
+                greaterThan(BigDecimal.ZERO));
+        verify(fixerioServiceProxy, never()).getRates(any(String.class));
+    }
+
+    @Test
+    public void testPositiveNumbers() throws IOException, UnknownCurrencyException {
+        CalculationCommand calculationCommand = new CalculationCommand(
+                new Value("usd", BigDecimal.valueOf(100)), new Value("cny", BigDecimal.valueOf(5)), Operation.ADD, "hkd");
+        assertThat(calculationCommandExecutionService.calculate(calculationCommand).getValue(), greaterThan(BigDecimal.ZERO));
+        verify(fixerioServiceProxy, times(2)).getRates(any(String.class));
```

Comment:
Here as opposed to previous comment the verification you do is not implied by the test name.